x-pack/plugins/upgrade_assistant/server/index.ts

```diff
@@ -14,9 +14,9 @@ export const plugin = (ctx: PluginInitializerContext) => {
 };
 
 export const config: PluginConfigDescriptor<Config> = {
+  deprecations: ({ deprecate }) => [deprecate('enabled', '8.0.0')],
```

Comment:
We'll need to add a `upgradeAssistant.ui.enabled` settings so we can advise users to use that setting instead.